chaos.py

```diff
@@ -77,14 +62,19 @@ def start_http_server():
     http_server_thread = threading.Thread(target=http_server)
     http_server_thread.start()
 
-if __name__ == "__main__":
-    log.info("starting up")
+def install_requirements():
+    """install or update requirements"""
+    os.system("pip install -r requirements.txt")
 
+if __name__ == "__main__":
+    logging.info("starting up and entering event loop")
+    
+    os.system("pkill chaos_server")
+    subprocess.Popen([sys.executable, "server.py"], cwd=join(THIS_DIR, "server"))
+    
     log.info("starting http server")
     start_http_server()
-
-    log.info("entering event loop")
->>>>>>> Add fortune server on port 80
+    
```

Comment:
It should be fixed, but there was a rebasing accident ( all these PR merged all the times made my fork collapse in merge errors xD ).

The history now is a little messy, If this is not voted I'll resubmit again from a cleaner branch.